optaplanner-core/src/main/java/org/optaplanner/core/config/localsearch/decider/acceptor/AcceptorConfig.java

```diff
@@ -74,6 +75,10 @@
     protected Integer stepCountingHillClimbingSize = null;
     protected StepCountingHillClimbingType stepCountingHillClimbingType = null;
 
+    protected String greatDelugeInitialWaterLevel = null;
+    protected Double greatDelugeWaterLevelIncrementRatio = null;
+    protected String greatDelugeWaterLevelIncrementScore = null;
```

Comment:
Nitpick: greatDelugeWaterLevelIncrementScore before greatDelugeWaterLevelIncrementRatio here too, also getter/setters and in the 2 methods that handle them.
Motivation:
- the ordering should be consistent in the code, so it's easier to spot when something is missing. Bugs are easier to spot this way.
- incrementScore is the simple one, so it should be first and incrementRatio is the more dynamic better one, so it should be second